src/settings.jl

```diff
@@ -98,6 +99,36 @@ function setting(k::Symbol)
     return getproperty(SETTINGS, k)
 end
 
+function _get_browser_cmd()
+    if Sys.iswindows()
+        return `cmd /c start`
+    elseif Sys.isapple()
+        return `open`
+    else
+        if is_wsl()
+            return `wslview`
+        else
+            return `xdg-open`
+        end
+    end
+end
+
+const _IS_WSL = Array{Union{Nothing,Bool},0}(undef)
```

Comment:
Cache the query so that future usage of `is_wsl()` can be free.

Reply:
I don't think it needs caching, it will only be called once per session typically anyway because the resulting command is cached.